### DISCUSSION TITLE
Heyo!

### DISCUSSION THREAD

**Discussion by LUISMAGAIVER341** [ravynsoft/ravynos#400] [Q&A]
Hi Ravyn devs!
I'm trying to install it, but i had 2 questions:

**1.  - Does it runs in 32 bits?
2.  - How to install?(LOL)**

**Comment by AssassinK786** [comment]
1. It is probably not possible
2. There is a procedure for it

**Comment by LUISMAGAIVER341** [comment]
Omg! nah... i was so animated!
